ddtrace/ddtrace.stub.php

```diff
@@ -0,0 +1,1051 @@
+     */
+    function curl_multi_exec_get_request_spans(&$array): void {}
+
+    /**
+     * Update a DogStatsD counter
+     *
+     * @param string $metric The metric name
+     * @param int $value The metric value
+     * @param array $tags A list of tags associated to the metric
+     */
+    function dogstatsd_count(string $metric, int $value, array $tags = []): void {}
+
+    /**
+     * Update a DogStatsD distribution
+     *
+     * @param string $metric The metric name
+     * @param float $value The metric value
+     * @param array $tags A list of tags associated to the metric
+     */
+    function dogstatsd_distribution(string $metric, float $value, array $tags = []): void {}
+
+    /**
+     * Update a DogStatsD gauge
+     *
+     * @param string $metric The metric name
+     * @param float $value The metric value
+     * @param array $tags A list of tags associated to the metric
+     */
+    function dogstatsd_gauge(string $metric, float $value, array $tags = []): void {}
+
+    /**
+     * Update a DogStatsD histogram
+     *
+     * @param string $metric The metric name
+     * @param float $value The metric value
+     * @param array $tags A list of tags associated to the metric
+     */
+    function dogstatsd_histogram(string $metric, float $value, array $tags = []): void {}
+
+    /**
+     * Update a DogStatsD set
+     *
+     * @param string $metric The metric name
+     * @param int $value The metric value
+     * @param array $tags A list of tags associated to the metric
+     */
+    function dogstatsd_set(string $metric, int $value, array $tags = []): void {}
+}
+
+namespace DDTrace\System {
+
+    /**
+     * Get the unique identifier of the container
+     *
+     * @return string|null The container id, or 'null' if no id was found
+     */
+    function container_id(): string|null {}
+}
+
+namespace DDTrace\Config {
+
+    /**
+     * Check if the app analytics of an app is enabled for a given integration
+     *
+     * @param string $integrationName The name of the integration (e.g., mysqli)
+     * @return bool The status of the app analytics of the integration
+     */
+    function integration_analytics_enabled(string $integrationName): bool {}
+
+    /**
+     * Check the app analytics sample rate of a given integration
+     *
+     * @param string $integrationName The name of the integration (e.g., mysqli)
+     * @return float The sample rate of the app analytics of the integration
+     */
+    function integration_analytics_sample_rate(string $integrationName): float {}
+}
+
+namespace DDTrace\UserRequest {
+    /**
+     * If there are any listeners of user request events.
+     * @return bool true iif there are any listeners
+     */
+    function has_listeners(): bool {}
+
+    /**
+     * Notifies the user request listeners of the start of a user request.
+     *
+     * @param \DDTrace\Span $span the span associated with this user request.
```

Comment:
The type of parameter declared in PHPDoc differs from one declared in signature. Could you please update the type to the same one?